docs/ai/how-to/app-service-aoai-auth.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Authenticate and Authorize App Service to Azure OpenAI using Microsoft Entra and the Semantic Kernel SDK"
+description: "Learn how to authenticate and authorize your app service application to an Azure OpenAI resource using Microsoft Entra managed identities and the Semantic Kernel SDK for .NET."
+author: haywoodsloan
+ms.topic: how-to
+ms.date: 04/19/2024
+zone_pivot_groups: azure-development-tool-set-two
+
+#customer intent: As a .NET developer, I want authenticate and authorize my App Service to Azure OpenAI using Microsoft Entra so that I can securely use AI in my .NET application.
+
+---
+
+# Authenticate and authorize App Service to Azure OpenAI using Microsoft Entra and the Semantic Kernel SDK
+
+This article demonstrates how to use Microsoft Entra managed identities to authenticate and authorize an App Service application to an Azure OpenAI resource.
```

Comment:
```suggestion
This article demonstrates how to use [Microsoft Entra managed identities](https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-id) to authenticate and authorize an App Service application to an Azure OpenAI resource.
```

Reply:
This doc is part of the other PR #40568 

I'll update it there.